source/blog/2016-11-02-ovirt-provider-ovn.md

```diff
@@ -178,6 +178,32 @@ Build the ovn rpms:
 
 The built rpms will reside here: `~/rpmbuild/RPMS/x86_64/`
 
+## Provider setup using the engine-setup script
+
+The provider can be set up on the engine host using the engine-setup tool.
+The engine-setup tool is part of oVirt engine installation and is located in:
+
+    /bin/engine-setup
+
+If used to set up ovirt-provider-ovn, engine-setup will perform the following tasks:
+
+* install ovs/ovn packages
+* install ovirt-provider-ovn packages
+* add and configure a default External Network OVN provider. The engine provider will be configured to connect to provider on localhost. The provider will also be configured with the user and password specified during the setup process.
+
+During the setup process, engine-setup script will ask the user several questions related to OVN:
```

Comment:
Maybe we can put this section after line 187 and change it a little?
Line 188 is the outcome